nuget/lib/dependabot/nuget/update_checker/compatibility_checker.rb

```diff
@@ -66,23 +66,31 @@ def project_tfms
       end
 
       def fetch_package_tfms(dependency_version)
-        nupkg_buffer = NupkgFetcher.fetch_nupkg_buffer(dependency_urls, dependency.name, dependency_version)
```

Comment:
The only change to this function is wrapping the body in `begin`/`end` so the result can be cached.  No change otherwise.